_notes/rft.md

```diff
@@ -0,0 +1,85 @@
+However, [Identified Gap]. 
+Ideally they should be able to [Ideal Outcome].
+
+Optional Details:
+[- Possible implementation ideas (e.g thoughts you have on how to achieve the ideal outcome)
+- Assessment criteria (e.g any tangible reflections you expect to see)
+- Engagement format (e.g looking for workshop/internship/lecture)
+- Engagement timeline (e.g available dates)
+- Etc]
+
+---
+SAMPLE RFT 1
+By: [Mr Jo]
+
+Students currently [are trying to launch their own startup ideas]. 
+However, [their copywriting skills are lacking, resulting in poor marketing outcomes]. 
+Ideally they should be able to [write compelling copy to showcase their ideas].
+
+Optional Details:
+[A 2h intro workshop during the September holidays would be great! My students should walk away being able to write better product titles.]
+
+---
+SAMPLE RFT 2
+By: [Mr Tan]
+
+Students currently [have 3d printers in the homeroom.]
+However, [they lack enough practice around CAD tooling to fully utilise the printers.] 
+Ideally, they should be able to [proficiently make and modify everyday items for printing after.]
+
+Optional Details:
+[Nil]
+
+---
+SAMPLE RFT 3
+By: [Ms Lim]
+
+Students currently [have to study "good governance" for social studies]. 
+However, [they find it really dry and just try to memorise the content, leading to poor understanding and application]. 
+Ideally they should be able to [learn about governance in a more engaging way].
+
+Optional Details:
+[It would be great if students get to learn more about other "Bad" government policies to contrast what "Good" governance mean.]
+
+------------------------------------------------------------------------------------
+# Request for Learners (RFL) Template
+
+I want to help students [long-term outcome] through [creative method].
+
+Optional Details:
+- This is in response to which RFT?
+- Please list your key lesson objectives
+- What is your exact lesson plan?
+- What is your exact activity plan?
+- What are some measurable metrics you would like to track?
+
+---
+SAMPLE RFLs:
+1. I want to help students [gain a deeper understanding for fractals and recursion] through [playing and programming simple graphics on the computer].
+2. I want to help students [refine their product launch copy] through [analysis how famous companies do it].
+3. I want to help students [learn more about how food is made] through [getting them to grow their own vege].
+4. I want to help students [better understand the importance of good governance] through [getting them to run a mini civilisation].
+
+
+------------------------------------------------------------------------------------
+# Mentor Profile Template
+
+
+{% include figure image_path="assets/lib/square_ikigai.png" alt="" %}
+
+1. Fullname?
+2. If asked to MENTOR students, which areas would you be willing to cover? (indicate your skills & interests)
+3. Beyond just mentoring, if you're asked to TEACH a class of students about anything you wish, what would it be?
```

Comment:
What is something you had always wanted to teach

 has something to it

or something you had always wanted others to learn about